src/components/modals/ReservedTokenReceiverModal.tsx

```diff
@@ -0,0 +1,104 @@
+import Modal from 'antd/lib/modal/Modal'
+import { t } from '@lingui/macro'
+import { FormInstance, Form, DatePicker } from 'antd'
+
+import { FormItems } from 'components/shared/formItems'
+
+import NumberSlider from 'components/shared/inputs/NumberSlider'
+import { useState } from 'react'
+
+type ModalMode = 'Add' | 'Edit' | undefined
+
+export default function ReservedTokenReceiverModal({
+  mode,
+  visible,
+  form,
+  onOk,
+  onCancel,
+  validateReservedTokenReceiver,
+  validateSlider,
+  reservedRate,
+}: {
+  mode: ModalMode
+  visible: boolean
+  form: FormInstance
+  onOk: VoidFunction
+  onCancel: VoidFunction
+  validateReservedTokenReceiver: VoidFunction
+  validateSlider: VoidFunction
+  reservedRate: number
+}) {
+  const [percent, setPercent] = useState<number>()
+
+  // Clarifies that the slider percentage is of the overall reserved allocation
+  // and shows the percentage of all newly minted tokens only if percent != 0
+  let sliderExtra = () =>
+    t`The percent this individual receives of the overall ${reservedRate}% 
+      reserved token allocation` +
+    `${
+      form.getFieldValue('percent')
+        ? ' ' +
+          t`(${(((reservedRate ?? 0) * 1.0 * (percent ?? 0)) / 100).toFixed(2)}%
```

Comment:
**suggestion**: perhaps would be more readable if we put this in a variable or 2.

```js
const sliderExtra = () => {
  const realTokenAllocation = (reservedRate ?? 0) * (percent ?? 0) * 1.0
  const realTokenAllocationPercent = (realTokenAllocation / 100).toFixed(2)
  const realTokenAllocationText = form.getFieldValue('percent')
    ? ` ${t`(${realTokenAllocationPercent}% of all newly minted tokens)}`}`
    : ''

  return t`The percent this address receives of the project's ${reservedRate}% 
reserved token allocation${realTokenAllocationText}.`
}
```

Reply:
Also, why do we need to `* 1.0`? Is this to convert to float or something?

Reply:
Yeah exactly

Reply:
`parseFloat` perhaps?